testsuite/MDAnalysisTests/coordinates/test_rdkit.py

```diff
@@ -65,14 +85,119 @@ def test_no_coordinates(self):
+        umol = sel.convert_to("RDKIT")
+        atom = umol.GetAtomWithIdx(atom_index)
+        mi = atom.GetMonomerInfo()
+
+        for mda_attr, rd_attr in RDATTRIBUTES.items():
+            rd_value = getattr(mi, "Get%s" % rd_attr)()
+            mda_value = getattr(sel, "%s" % mda_attr)[atom_index]
+            if mda_attr == "names":
+                rd_value = rd_value.strip()
+            assert rd_value == mda_value
+
+    def test_identical_topology_mol2(self, mol2):
+        """Check stereochemistry on atoms and bonds (but not yet)"""
+        rdmol = mol2_mol()
+        umol = mol2.atoms.convert_to("RDKIT")
+        assert rdmol.HasSubstructMatch(umol, useChirality=False)
+        assert umol.HasSubstructMatch(rdmol, useChirality=False)
+
+    def test_identical_topology(self):
+        rdmol = smiles_mol()
+        u = mda.Universe(rdmol)
+        umol = u.atoms.convert_to("RDKIT")
+        assert rdmol.HasSubstructMatch(umol) and umol.HasSubstructMatch(rdmol)
+
+    def test_raise_requires_elements(self):
+        u = mda.Universe(mol2_molecule)
+        with pytest.raises(AttributeError) as e:
+            u.atoms.convert_to("RDKIT")
+            assert "`elements` attribute is required for the RDKitConverter" in str(
+                e.value)
+
+    def test_warn_guess_bonds(self, pdb):
+        pdb.delete_bonds(pdb.bonds)
+        ag = pdb.select_atoms("resnum 101 and segid A")
+        pdb.delete_bonds(ag.bonds)
+        with warnings.catch_warnings(record=True) as w:
+            # Cause all warnings to always be triggered.
+            warnings.simplefilter("always")
+            # trigger warning
+            ag.convert_to("RDKIT")
+            assert len(w) == 1
+            assert "No `bonds` attribute in this AtomGroup" in str(
+                w[-1].message)
+
+    @pytest.mark.parametrize("attr, value, expected", [
+        ("names", "C1", " C1 "),
+        ("names", "C12", " C12"),
+        ("names", "Cl1", "Cl1 "),
+        ("altLocs", "A", "A"),
+        ("chainIDs", "B", "B"),
+        ("icodes", "C", "C"),
+        ("occupancies", 0.5, 0.5),
+        ("resnames", "LIG", "LIG"),
+        ("resids", 123, 123),
+        ("segindices", 1, 1),
+        ("tempfactors", 0.8, 0.8),
+    ])
+    def test_add_mda_attr_to_rdkit(self, attr, value, expected):
+        mi = Chem.AtomPDBResidueInfo()
+        _add_mda_attr_to_rdkit(attr, value, mi)
+        rdvalue = getattr(mi, "Get%s" % RDATTRIBUTES[attr])()
+        assert rdvalue == expected
+
+    @pytest.mark.parametrize("idx", [0, 10, 42])
+    def test_other_attributes(self, mol2, idx):
+        mol = mol2.atoms.convert_to("RDKIT")
+        rdprops = mol.GetAtomWithIdx(idx).GetPropsAsDict()
+        for prop in ["charge", "segid", "type"]:
+            rdprop = rdprops["_MDAnalysis_%s" % prop]
+            mdaprop = getattr(mol2.atoms[idx], prop)
+            assert rdprop == mdaprop
+
+
+@pytest.mark.skipif(rdkit_installed == True, reason="test minimal dependency")
```

Comment:
One other possibility is to move this up and use block_import to allow to fake a minimal dependency.